haystack/modeling/training/base.py

```diff
@@ -36,7 +36,7 @@
 class EarlyStopping:
     """
     Can be used to control early stopping with a Trainer class. Any object can be used instead which
-    implements the method check_stopping and and provides the attribute save_dir
+    implements the method check_stopping and provides the attribute save_dir
```

Comment:
I'm not sure I understand this. You can use this class but you can use any object instead?

Reply:
I think they mean that a user could create their own `EarlyStopping` class (e.g. `CustomEarlyStopping`) as long as that new class implements a `check_stopping` method and has the attribute `save_dir`. I'm not really sure if this document string is needed at the moment so I'd be happy to remove it if is just causing confusion.

Reply:
@agnieszka-m I've updated the docstring to hopefully be more clear about what was meant.